doc/user-guide.md

```diff
@@ -401,19 +399,17 @@ func ExampleStorageClient() {
     "127.0.0.1:8082",
   }
 
-  mgr, err := NewManager(WithNodeList(addrs), WithDialTimeout(50*time.Millisecond), WithGrpcDialOptions(
-    grpc.WithBlock(),
-    grpc.WithInsecure(),
+  mgr := NewManager(
+    WithDialTimeout(50*time.Millisecond),
```

Comment:
should be `gorums.WithDialTimeout`, `gorums.WithGrpcDialOptions` and `gorums.WithNodeList`